src/Microsoft.OData.Core/ODataSimplifiedOptions.cs

```diff
@@ -28,16 +28,15 @@ public ODataSimplifiedOptions(ODataVersion? version)
         {
             this.EnableParsingKeyAsSegmentUrl = true;
             this.EnableWritingKeyAsSegment = false;
+            this.EnableReadingKeyAsSegment = false;

```

Comment:
Question: this will change reader's EnableReadingKeyAsSegment default to false for V401 ? Just want to make sure this is intentional.

Reply:
I had mistakenly changed it to true in a previous PR to this release, thinking it allowed the reader to recognize key-as-segment.  In fact, it means that generated keys use key-as-segment, which should be off by default, so I fixed it here.  Thanks for checking!